public/pages/Indices/containers/IndicesActions/index.tsx

```diff
@@ -302,7 +302,7 @@ export default function IndicesActions(props: IndicesActionsProps) {
                     },
                     {
                       name: "Shrink",
-                      disabled: !selectedItems.length || selectedItems.length != 1,
+                      disabled: !selectedItems.length || selectedItems.length > 1 || selectedItems[0].data_stream !== null,
```

Comment:
Is it possible that data_stream is ""?

Reply:
It's impossible,  we use `Get _data_streams API` to get all data_streams firstly and then construct a index->data_stream mapping, if the index is not a backend index of a data_stream then the `data_stream` field is `null`:
https://github.com/opensearch-project/index-management-dashboards-plugin/blob/225478710cf544ca898ac9a866583d8a5bd55cd5/server/services/IndexService.ts#L103

Reply:
Although it's impossible that the `data_stream` field is "", but I noticed that we set `data_stream` to "" in many unit test cases, and we should not disable shrink or split button in these test cases, so I think we should use `!!selectedItems[0].data_stream` instead of `selectedItems[0].data_stream !== null`, that means if one index's `data_stream` field is "" or `undefined` or `null`, the index is not a backend index of a data_stream definitely, so the split or shrink button cannot be disabled.